package/kernel/mac80211/broadcom.mk

```diff
@@ -451,6 +451,7 @@ define KernelPackage/brcmfmac/config
 		bool "Enable SDIO bus interface support"
 		default y if TARGET_bcm27xx
 		default y if TARGET_sunxi
+		default y if TARGET_imx6_DEVICE_cubox-i
```

Comment:
I don't think this kind of package selection will work per device, but only per target.